src/nvim/api/dispatch_deprecated.lua

```diff
@@ -7,6 +7,7 @@ local deprecated_aliases = {
   nvim_buf_get_number="buffer_get_number",
   nvim_buf_get_option="buffer_get_option",
   nvim_buf_get_var="buffer_get_var",
+  nvim_buf_live_updates="buffer_live_updates",
```

Comment:
if you use master version of neovim python-client (release will be made soon) this will not be needed.